### ISSUE TITLE
Accessibility: Notification announcements missing

### ISSUE THREAD

**Issue by daniellekirkwood** [element-hq/element-web#30395]
### Steps to reproduce

We've had feedback from a screenreader user that our web & desktop notifications are not satisfactory and could do wtih some improvement. One appears the be a regression.

Suspected regression: 
- Notifications don't show up at call. 
- Element still makes a sound so "we're not entirely lost"
- The version prior said "New notifications from im.riot.app"

Other issue raised:
- If messages are sent to the room you are in, notifications are not sent 
- Users also don't get a screenreader announcement either
- This means you have to manually check the timeline for new messages

We've had this feedback before and I believe ```aira-polite``` was suggested as a fix

#### What did you expect?
New messages should be told via screenreader so users know whats happening

#### What happened instead?
No announcement

### Operating system

Windows (might be mac too, not tested)

### Will you send logs?

No
